### DISCUSSION TITLE
Problema nuova versione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#415] [Richieste di aiuto]
Persiste il problema legato al nome del file del modello che presenta il menù a tendina vuoto

**Comment by Lucaccc** [comment]
I modelli devi personalizzarli e caricarli tu.
Segui la guida pubblicata.
La guida è qui: https://iisgiua.github.io/giuaschool-docs/task-request.html#uso-dei-modelli-template

**Comment by sempliciniluigi89** [comment]
@Lucaccc Mi potrebbe aiutare lei dal momento che già lo ha fatto?Grazie ,mi contatti pure all'email : sempliciniluigi89@gmail.com
